src/benchmarks/micro/libraries/System.Text.Json/Utf8JsonReader/Perf.Get.cs

```diff
@@ -23,214 +25,189 @@ private static byte[] GetJsonBytes<T>(T elem)
             return Encoding.UTF8.GetBytes(elem.ToString());
         }
 
-        [Benchmark]
-        public byte GetByte()
+        [Benchmark(OperationsPerInvoke = OperationsPerBenchmark)]
+        public void GetBoolean()
         {
-            byte result = 0;
-            var reader = new Utf8JsonReader(_jsonIntegerNumberBytes);
+            var reader = new Utf8JsonReader(_jsonFalseBytes);
             reader.Read();
 
-            for (int i = 0; i < 100; i++)
+            for (int i = 0; i < OperationsPerBenchmark; i++)
             {
-                result += reader.GetByte();
+                reader.GetBoolean();
```

Comment:
We need to consume the result to make sure that some smart compilers (like the Mono AOT compiler) are not going to treat it as dead code and remove it:

```cs
public bool GetBoolean()
{
    var reader = new Utf8JsonReader(_jsonFalseBytes);
    reader.Read();

    bool result = false;
    for (int i = 0; i < OperationsPerBenchmark; i++)
    {
        result ^= reader.GetBoolean();
    }
    return result;
}
```

https://github.com/dotnet/performance/blob/main/docs/microbenchmark-design-guidelines.md#dead-code-elimination

Reply:
I'd be surprised if any compiler applied DCE to method calls, since it would need to be able to reason about mutations happening to the `reader` instance. But maybe some compilers are smarter than I would have thought 🤷 

In this case I think it makes sense to remove aggregation: these really are micro-level benchmarks, so I would anticipate that the added instructions contribute to noise.